src/7-to-8/summary.rst

```diff
@@ -227,116 +124,20 @@ Optional and Expected Task Outputs
 
    * :ref:`728.suicide_triggers`
 
```

Comment:
Just above this line, suicide triggers can just be listed along with expected and optional outputs. They're not "more major" than those.

Reply:
Throughout the summary, the see also sections have the format:

```
User Guide
- links
- links
Major Changes
- links
- links
```
I have kept this format, although I think you raise a really good point that it looks as if suicide triggers are "more major" than other links.
Would it be worth reconsidering the format of the `See also` sections and just have them as a plain list, rather than separating them into user guide links and major changes links?

Reply:
Yeah I'd prefer a single list. The links can self-declare their target location, e.g. "User Guide: Suicide Triggers".